### PR TITLE
AT_18.01_005| Verify access to user ID and status

### PR DESCRIPTION
**Pull request by matvlesia**
https://trello.com/c/ROOWaP5b/1073-at1801005-verify-access-to-user-id-and-status-though-users-profile-page

### CODE REVIEW

**Comment by KaterinaNa** [comment]
Обратите внимание на заполнение describe, 
не ясно, зачем нужны let

File: cypress/e2e/profilePage.cy.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /// <reference types="cypress"/>
 
 describe("Profile Page", () => {
+  const USERNAME = Cypress.env('local.admin.username');
```
Comment (Rom-T) [inline cypress/e2e/profilePage.cy.js:4]: Such variables are called global and are taken out before describe. You can do it later with refactoring.

Comment (matvlesia) [reply cypress/e2e/profilePage.cy.js]: Thank you, I moved it in front of describe

**Review by TatiChap — APPROVED** [review]
LGTM

**Pull request merged**